### PR TITLE
Update bmm-sdk

### LINKED ISSUE #85: swagger fixes (e.g. IDocument)

### PR DESCRIPTION
**Pull request by SimonSimCity**
Fixes #85

### CODE REVIEW

**Review by kkuepper — APPROVED** [review]
looks good to me

File: composables/album.ts [hunk 1]
```diff
@@ -1,25 +1,17 @@
...
 }
 
-/**
- * Get playlist with the specified id
- */
 export function useAlbum(options: UseAlbumOptions) {
   const { id } = options;
 
-  return useLazyAsyncData<AlbumModel>(`album-${id}`, () =>
+  return useLazyAsyncData(`album-${id}`, () =>
```
Comment (sifferhans) [inline composables/album.ts:10]: Are the types (AlbumModel) inferred here?

**Review by sifferhans — COMMENTED** [review]
I had just one question, but looks good otherwise

File: composables/album.ts (hunk 1 above)
Comment (SimonSimCity) [reply composables/album.ts:10]: Yes, they are 👍

**Pull request merged**